migration-roadmap/radio.md

```diff
@@ -0,0 +1,236 @@
+    />
+    <span class="spectrum-Radio-button spectrum-Radio-button--sizeS"></span>
+    <label
+        class="spectrum-Radio-label spectrum-Radio-label--sizeS"
+        for="radio-input-123"
+    >
+        Radio Button Label
+    </label>
+</div>
+```
+
+</details>
+
+<details>
+<summary>Diff: Legacy (CSS main) → Spectrum 2 (CSS spectrum-two)</summary>
+
+**No differences found between main and spectrum-two branches.**
+
+</details>
+
+### CSS => SWC mapping
+
+| CSS selector                                                                                         | Attribute or slot                        | Status           |
+| ---------------------------------------------------------------------------------------------------- | ---------------------------------------- | ---------------- |
+| `.spectrum-Radio`                                                                                    | `:host`                                  | Implemented      |
+| `.spectrum-Radio--emphasized`                                                                        | `emphasized` attribute                   | Implemented      |
+| `.spectrum-Radio--sizeS`                                                                             | `size="s"` attribute                     | Implemented      |
+| `.spectrum-Radio--sizeL`                                                                             | `size="l"` attribute                     | Implemented      |
+| `.spectrum-Radio--sizeXL`                                                                            | `size="xl"` attribute                    | Implemented      |
+| `.spectrum-Radio-input`                                                                              | Internal radio input element, `#input`   | Implemented      |
+| `.spectrum-Radio-input:checked`                                                                      | `checked` attribute                      | Implemented      |
+| `.spectrum-Radio-input:disabled`                                                                     | `disabled` attribute                     | Implemented      |
+| `.spectrum-Radio-button`                                                                             | Internal radio button element, `#button` | Implemented      |
+| `.spectrum-Radio-label`                                                                              | Default slot content                     | Implemented      |
+| `.spectrum-Radio-label:lang(ja)`, `.spectrum-Radio-label:lang(ko)`, `.spectrum-Radio-label:lang(zh)` | Language-specific styling                | Implemented      |
+| `.spectrum-Radio.is-readOnly`                                                                        | `readonly` attribute                     | Implemented      |
+|                                                                                                      | `invalid` attribute                      | Missing from CSS |
```

Comment:
This is similar to the checkbox, where I think instead of putting `invalid` on the radio, we pass that arg/property to the help text instead. So radio itself doesn't really have an invalid state- the radio _group_ has an invalid state that is shown via negative help text.